packages/protocol-contracts/contracts/ZetaTokenConsumerUniV3.strategy.sol

```diff
@@ -59,6 +63,13 @@ contract ZetaTokenConsumerUniV3 is ZetaTokenConsumer, ZetaTokenConsumerUniV3Erro
         tokenPoolFee = tokenPoolFee_;
     }
 
+    modifier noReentrant() {
```

Comment:
Can we call it `nonReentrant`? Like [OZ](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/ReentrancyGuard.sol).